vulnerabilities/pipelines/v2_importers/npm_importer.py

```diff
@@ -130,6 +134,7 @@ def to_advisory_data(self, file: Path) -> Iterable[AdvisoryData]:
             references_v2=references,
             severities=severities,
             url=f"https://github.com/nodejs/security-wg/blob/main/vuln/npm/{id}.json",
+            original_advisory_text=advisory_text or json.dumps(data, indent=2, ensure_ascii=False),
```

Comment:
Isn't `advisory_text` already a JSON dump? 

> advisory_text or json.dumps(data, indent=2, ensure_ascii=False)

This seems redundant, if we don't have an `advisory_text` how can we have `data`.